package.json

```diff
@@ -6,14 +6,18 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "cypress run",
+    "cypress": "cypress open"
   },
   "dependencies": {
     "next": "12.2.2",
     "react": "18.2.0",
     "react-dom": "18.2.0"
   },
   "devDependencies": {
+    "babel-plugin-styled-components": "^2.0.7",
```

Comment:
✨ its good to get this set up early within the devDependencies, brilliant work 🤩 ⭐